<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-15T22:20:21 IST" name="DemoWebShopSuite" finished-at="2025-08-15T22:20:35 IST" duration-ms="14186">
    <groups>
    </groups>
    <test started-at="2025-08-15T22:20:21 IST" name="DemoWebShopTest" finished-at="2025-08-15T22:20:35 IST" duration-ms="14186">
      <class name="com.web.DemoWebShopTest">
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:com.web.DemoWebShopTest@3eb25e1a]" started-at="2025-08-15T22:20:21 IST" name="setUp" finished-at="2025-08-15T22:20:30 IST" duration-ms="8750" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://demowebshop.tricentis.com/]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyHomePageTitle()[pri:1, instance:com.web.DemoWebShopTest@3eb25e1a]" started-at="2025-08-15T22:20:30 IST" name="verifyHomePageTitle" finished-at="2025-08-15T22:20:30 IST" duration-ms="15" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHomePageTitle -->
        <test-method signature="navigateToLoginPage()[pri:2, instance:com.web.DemoWebShopTest@3eb25e1a]" started-at="2025-08-15T22:20:30 IST" name="navigateToLoginPage" finished-at="2025-08-15T22:20:33 IST" duration-ms="2585" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigateToLoginPage -->
        <test-method signature="loginTest()[pri:3, instance:com.web.DemoWebShopTest@3eb25e1a]" started-at="2025-08-15T22:20:33 IST" name="loginTest" finished-at="2025-08-15T22:20:35 IST" depends-on-methods="com.web.DemoWebShopTest.navigateToLoginPage" duration-ms="2564" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login failed or 'Log out' link not found expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login failed or 'Log out' link not found expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at com.web.DemoWebShopTest.loginTest(DemoWebShopTest.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method signature="verifyProductList()[pri:4, instance:com.web.DemoWebShopTest@3eb25e1a]" started-at="2025-08-15T22:20:35 IST" name="verifyProductList" finished-at="2025-08-15T22:20:35 IST" depends-on-methods="com.web.DemoWebShopTest.loginTest" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.web.DemoWebShopTest.verifyProductList() on instance com.web.DemoWebShopTest@3eb25e1a depends on not successfully finished methods 
[com.web.DemoWebShopTest.loginTest() on instance com.web.DemoWebShopTest@3eb25e1a]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.web.DemoWebShopTest.verifyProductList() on instance com.web.DemoWebShopTest@3eb25e1a depends on not successfully finished methods 
[com.web.DemoWebShopTest.loginTest() on instance com.web.DemoWebShopTest@3eb25e1a]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:159)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyProductList -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.web.DemoWebShopTest@3eb25e1a]" started-at="2025-08-15T22:20:35 IST" name="tearDown" finished-at="2025-08-15T22:20:35 IST" duration-ms="211" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.web.DemoWebShopTest -->
    </test> <!-- DemoWebShopTest -->
  </suite> <!-- DemoWebShopSuite -->
</testng-results>
